GitHubContentUtility/Operations/BlobContentWriter.cs

```diff
@@ -0,0 +1,175 @@
+                                                appConfig.GitHubRepoName,
+                                                blob);
+
+                    // Create new Tree
+                    var tree = new NewTree { BaseTree = latestCommit.Tree.Sha };
+
+                    var treeMode = (int)appConfig.TreeItemMode;
+
+                    // Add items based on blobs
+                    tree.Tree.Add(new NewTreeItem
+                    {
+                        Path = appConfig.FileContentPath,
+                                    Mode = treeMode.ToString(),
+                                    Type = TreeType.Blob,
+                                    Sha = blobRef.Sha
+                    });
+
+                    var newTree = await finalClient.Git.Tree.Create(appConfig.GitHubOrganization,
+                                    appConfig.GitHubRepoName,
+                                    tree);
+
+                    // Create a commit
+                    var newCommit = new NewCommit(appConfig.CommitMessage,
+                                        newTree.Sha,
+                                        workingReference.Object.Sha);
+
+                    var commit = await finalClient.Git.Commit.Create(appConfig.GitHubOrganization,
+                                    appConfig.GitHubRepoName,
+                                    newCommit);
+
+                    // Push the commit
+                    await finalClient.Git.Reference.Update(appConfig.GitHubOrganization,
+                        appConfig.GitHubRepoName,
+                        workingBranch.Ref,
+                        new ReferenceUpdate(commit.Sha));
+                }
+
+                // Create a PR
+                var pullRequest =
+                    await finalClient.Repository.PullRequest.Create(appConfig.GitHubOrganization,
+                        appConfig.GitHubRepoName,
+                        new NewPullRequest(appConfig.PullRequestTitle,
+                            appConfig.WorkingBranch,
+                            appConfig.ReferenceBranch)
+                            { Body = appConfig.PullRequestBody });
+
+
+                // Add PR reviewers
+                if (appConfig.Reviewers != null)
+                {
+                    var reviewersResult = await finalClient.Repository.PullRequest.ReviewRequest.Create(appConfig.GitHubOrganization,
+                    appConfig.GitHubRepoName,
+                    pullRequest.Number,
+                    new PullRequestReviewRequest(appConfig.Reviewers.AsReadOnly(), null));
+                }
+
+                var issueUpdate = new IssueUpdate();
+
+                // Add PR assignee
+                if (appConfig.PullRequestAssignee != null)
+                {
+                    foreach(var assignee in appConfig.PullRequestAssignee)
+                    {
+                        issueUpdate.AddAssignee(assignee);
+                    }
+                }
```

Comment:
nit:
```suggestion
                appConfig.PullRequestAssignee?.ForEach(assignee => issueUpdate.AddAssignee(assignee));
```

Reply:
So cool 😎 Thanks!
I already updated the property names (pluralisation) so I have ported your suggestion directly into my editor.